tutorials/lava/lib/dl/slayer/nmnist/train.ipynb

```diff
@@ -3,7 +3,11 @@
   {
```

Comment:
I don't think there are any changes except your local run metadata. So this can perhaps be removed from the PR

Reply:
My bad, I have deleted this